app/uk/gov/hmrc/apinotificationpull/controllers/EnhancedNotificationsController.scala

```diff
@@ -105,8 +105,8 @@ class EnhancedNotificationsController @Inject()(enhancedApiNotificationQueueServ
       enhancedApiNotificationQueueService.getNotificationBy(notificationId, notificationStatus)
         .map {
           case Right(n) =>
-            val conversationId = n.headers.get("X-Conversation-ID")
-            logger.info(s"Notification found for conversationId id: $conversationId with notification id: $notificationId")
+            val conversationId = n.headers.getOrElse("x-conversation-id", "None")
```

Comment:
Extract this into a val? Or another class with all header keys?

Reply:
Added constant for header name